doc/source/news.rst

```diff
@@ -7,6 +7,238 @@
 News
 ====
 
+.. _release-11-0-1:
+
+Release 11.0.1 - 2021-03-29
+---------------------------
+
+Improvements
+^^^^^^^^^^^^
+
+* [:doc:`/install/debian`] Added support for a ARM64 package.
+
+* [:doc:`reference/commands/select`] Added support for getting a reference destination data via index column when we output a result.
+
+  * Until now, Groonga return involuntary value when we specify output value like "index_column.xxx".
+    However, we can get a reference destination data via index column as below since this release.
+
+    .. code-block::
+
+      table_create Products TABLE_PAT_KEY ShortText
+
+      table_create Purchases TABLE_NO_KEY
+      column_create Purchases product COLUMN_SCALAR Products
+      column_create Purchases tag COLUMN_SCALAR ShortText
+
+      column_create Products purchases COLUMN_INDEX Purchases product
+
+      load --table Products
+      [
+      {"_key": "apple"},
+      {"_key": "banana"},
+      {"_key": "cacao"}
+      ]
+
+      load --table Purchases
+      [
+      {"product": "apple",  "tag": "man"},
+      {"product": "banana", "tag": "man"},
+      {"product": "cacao",  "tag": "woman"},
+      {"product": "apple",  "tag": "many"}
+      ]
+
+      select Products \
+        --columns[tags].stage output \
+        --columns[tags].flags COLUMN_VECTOR \
+        --columns[tags].type ShortText \
+        --columns[tags].value purchases.tag \
+        --output_columns _key,tags
+
+* [:doc:`reference/commands/select`] Added support for specifying index column directly as a part of nested index.
+
+  * Until now, we can only specify which column to use like "index_column.source_column" when we search.
```

Comment:
Using relative pronoun "which" here?
If so, 

Until now, we can only specify **a column which uses** like "index_column.source_column" when we search.

is correct sentence, I guess.
If we use relative pronoun, we can split two sentences:

Until now, we can only specify a column.
A column uses like "index_column.source_column" when we search.

Relative pronoun concatenate these two sentences into one sentence.